### PR TITLE
feat(observable-md): Add new Observable Markdown Widget

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/observable-md/index.html [hunk 1]
```diff
@@ -0,0 +1,163 @@
+<!doctype html>
```
Comment (jbrundage) [inline packages/observable-md/index.html]: I like the idea of having an `index.html` in a package directory, but this is not standard for the project. Should we aim to have an `index.html` in each package showcasing some simple usage?

File: packages/loader/src/meta.ts [hunk 2]
```diff
@@ -42,13 +42,19 @@ export const npmPackages = {
     "dagre": "dagre/dist/dagre",
     "@deck.gl/core": "@deck.gl/core/dist",
     "@deck.gl/layers": "@deck.gl/layers/dist",
+    "@observablehq/parser": "@observablehq/parser/dist/parser.min",
+    "@observablehq/runtime": "@observablehq/runtime/dist/runtime.umd",
+    "@observablehq/inspector/dist/inspector.css": "@observablehq/inspector/dist/inspector.css",
+    "acorn": "acorn/dist/acorn",
+    "acorn-walk": "acorn-walk/dist/walk",
```
Comment (jbrundage) [inline packages/loader/src/meta.ts]: What will we use `acorn` for?

Comment (GordonSmith) [reply packages/loader/src/meta.ts]: We don't but its pulled in by observable...

File: packages/observable-md/index.html (hunk 1 above)
Comment (GordonSmith) [reply packages/observable-md/index.html]: I see an index.html as being part of docs, so the more the better...

**Pull request merged**